GiTinder.Tests/Models/SettingsTest.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Xunit;
+
+namespace GiTinder.Tests.Models
+{
+    public class SettingsTest
+    {
+
+        [Fact]
+        public void CanCreateSettingsWithNullUserName()
+        {
+            var settings = SettingsFactory.CreateSettings();
+            Assert.True(settings.UserName == null);
+        }
+
+        [Fact]
+        public void CanSetUserNameWithString()
+        {
+            var settings = SettingsFactory.CreateSettings();
+            settings.UserName = "Pablos";
+            Assert.Equal("Pablos", settings.UserName);
+        }
+
+
+
+
+
+        [Fact]
+        public void CanCreateSettingsWithGivenNameAndDefaultSettings()
+        {
+
+            var testSettings = SettingsFactory.CreateSettingsWithGivenUserNameTest_TomekAndDefaultSettings();
+
+
+            Assert.True(testSettings.UserName == "Test_Tomek");
+            Assert.True(testSettings.EnableNotification == true);
+            Assert.True(testSettings.MaxDistanceInKm == 160);
+
+            Assert.False(testSettings.MaxDistanceInKm == 1);
+            Assert.True(testSettings.PreferredLanguages == "English");
+            Assert.False(testSettings.PreferredLanguages == "Czech");
+            
+        }
```

Comment:
@FilipTeply this is also just testing the language, unnecessary

Reply:
I removed the "Czech" testing line.